lib/gpu/render_pass.h

```diff
@@ -252,6 +252,11 @@ extern void InternalFlutterGpu_RenderPass_SetCullMode(
     int cull_mode);
 
 FLUTTER_GPU_EXPORT
+
+extern void InternalFlutterGpu_RenderPass_SetPrimitiveType(
+    flutter::gpu::RenderPass* wrapper,
+    int primitive_type);
+
 extern void InternalFlutterGpu_RenderPass_SetWindingOrder(
```

Comment:
The tests are failing because `InternalFlutterGpu_RenderPass_SetWindingOrder` is no longer being exported.

```
00:00 [32m+19[0m[31m -1[0m: Drawing respects cull mode [1m[31m[E][0m[0m

  Invalid argument(s): Couldn't resolve native function 'InternalFlutterGpu_RenderPass_SetWindingOrder' in 'file:///b/s/w/ir/cache/builder/src/flutter/lib/gpu/lib/gpu.dart' : No asset with id 'file:///b/s/w/ir/cache/builder/src/flutter/lib/gpu/lib/gpu.dart' found. No available native assets. Attempted to fallback to process lookup. /b/s/w/ir/cache/builder/src/out/ci/host_debug/exe.unstripped/flutter_tester: undefined symbol: InternalFlutterGpu_RenderPass_SetWindingOrder.
  

  dart:ffi-patch/ffi_patch.dart                      Native._ffi_resolver.#ffiClosure0
  dart:ffi-patch/ffi_patch.dart 1578:20              Native._ffi_resolver_function
  flutter/lib/gpu/lib/src/render_pass.dart           RenderPass.__setWindingOrder$Method$FfiNative
  flutter/lib/gpu/lib/src/render_pass.dart 434:17    RenderPass._setWindingOrder
  flutter/lib/gpu/lib/src/render_pass.dart 271:5     RenderPass.setWindingOrder
  flutter/testing/dart/gpu_test.dart 493:22          main.<fn>
  package:test_api/src/backend/declarer.dart 229:19  Declarer.test.<fn>.<fn>
  ===== asynchronous gap ===========================
  package:test_api/src/backend/declarer.dart 227:7   Declarer.test.<fn>
  ===== asynchronous gap ===========================
  package:test_api/src/backend/invoker.dart 258:9    Invoker._waitForOutstandingCallbacks.<fn>
```

All of these symbols need to be prefixed with the `FLUTTER_GPU_EXPORT` macro.

Reply:
Added. Thanks!